roles/django/defaults/main.yml

```diff
@@ -0,0 +1,30 @@
+---
+# main config
+django_app: myapp
+django_app_path: /srv/www/{{ django_app }}
+django_user: "{{ ansible_user }}"
```

Comment:
`ansible_user` might be pulsys rather than conan — is this so you don't have to depend on / be aware of `deploy_user` ? Is your intention that we override it? And if so, can you set our standard as a default in the shared variables file?

Reply:
yes, yes, and yes 🙂